NEWS.md

```diff
@@ -1,3 +1,9 @@
+# hubAdmin (development)
```

Comment:
At the moment, I'm going to decline the suggestion. I don't think it's that useful since this is going to be changing up until the release. Keeping the number off also allows us to increment the dev component with each PR without having to change it in two places.

Reply:
Totally fine by me